server/src/main/java/com/netflix/conductor/server/ServerModule.java

```diff
@@ -114,6 +116,8 @@ protected void configure() {
 		} else {
 			bind(ExternalPayloadStorage.class).to(DummyPayloadStorage.class);
 		}
+		
+		bind(WorkflowSweeper.class).asEagerSingleton();
```

Comment:
Why are we adding this binding? I don't believe our work on this should affect the core module loading in any way at all.

Reply:
For timeouts to work, we need this binding. WorkflowSweeper is the background thread that sweeps _deciderQueue regularly to check from Timeouts. I am not sure why this was removed in the original implementation. May be @v1r3n has the reason for omitting this change

Reply:
I have reverted the ui/package-lock.json from origin/dev. I am not sure why this file shows as change. I could have entered a space by mistake